### DISCUSSION TITLE
Any progress of supporting `rotateX` or `rotateY`? @adamwathan

### DISCUSSION THREAD

**Discussion by boosting-my-adrenaline** [tailwindlabs/tailwindcss#7009] [General]
Any progress of supporting `rotateX` or `rotateY` directly in tailwind?

**Comment by sambauers** [comment]
I made this recently. It’s early days, but I think it has potential to fill this gap and others around 3D transforms.

https://github.com/sambauers/tailwindcss-3d

A bit lacking in documentation, but should make sense to most folks familiar with Tailwind transforms.
